### ISSUE TITLE
When providing multiple paths in, separate them by "," instead of "%%"

### ISSUE THREAD

**Issue by dukeofharen** [dukeofharen/httplaceholder#53]
This was an ugly workaround I've built a few years ago. When you want to provide multiple paths for the inputFile parameter, you do it like this:

```bash
httplaceholder --inputFile /var/stubs1%%/var/stubs2
```

I've did it like this because PowerShell didn't allow comma's, but it will allow comma's if the inputFile string is wrapped with double quotes. So for PowerShell, this doesn't work:

```bash
httplaceholder --inputFile /var/stubs1,/var/stubs2
```

But this does:

```bash
httplaceholder --inputFile "/var/stubs1,/var/stubs2"
```

In all other terminals, using comma's without quotes seems to work just fine.

Long story short, this will fix a very ugly workaround I've built some time ago. Using "%%" will still work though for backwards compatability.
